.editorconfig

```diff
@@ -0,0 +1,24 @@
+﻿# EditorConfig is awesome:http://EditorConfig.org
```

Comment:
@muratg is this something we want in all the projects?

@natemcmaster Which editors understand this? Does it require plugins?